### DISCUSSION TITLE
Displays

### DISCUSSION THREAD

**Discussion by yobanyvrod** [CreativeMD/LittleTiles#961] [Ideas]
### A display that receives links, images, displays colours, text and plays audio*1.
This is my idea for how I imagine a display would be implemented:
<details><summary>*1</summary>
<p>

separate audio block/function could be implemented along with the display

</p>
</details> 

This could be done a couple of ways 
1:
One would make a shape they would like to be a display.
<details><summary>Image</summary>
<p>

![image](https://github.com/CreativeMD/LittleTiles/assets/66252573/60a08bec-62c4-45bb-bf5c-5e262d71e401)

</p>
</details> 

Next they would make it a blueprint and would select display in the logical component dropdown menu.
<details><summary>Image</summary>
<p>

![image](https://github.com/CreativeMD/LittleTiles/assets/66252573/2bf39bb9-b5d3-49cf-8d9b-eacd71c97404)

</p>
</details> 
2:
A display block could be added that would mimic the function of the first example.
One could then make a shape they would like for their display and the display could then be right clicked to turn on/off
and Ctrl+C could be pressed to modify the settings.

_______________

The display could then be modified to the user's desires (Aspect ratio, number of pixels, Hz, colour augmentation, display type(image, video, webm, webp) responsive touch screen with external code implementation abilities(a little like the Computer Craft mod))

The display could then fetch links from online streaming/video platforms e.g., YouTube, Twitch, Kick. Along with still images from Google/any other image library.
The display could also display solid colour; text; data based on an input it has been given from another LittleTiles structure.

Display blocks could ideally be moved and display a live feed of online content, with touchscreen capability to operate like a computer inside of Minecraft.

As always, Thank you in advance for reading and considering my idea proposal.

**Comment by CreativeMD** [comment]
This sounds quite familiar to LittleFrames. They can even be controlled via signals, but only paused and played again. Not sure what else to add. If you have any ideas let me know. But at the moment your idea seems to be way to difficult for me to implement at the moment, sorry.

Also very sorry to not have responded to your suggestion for half a year.
